xml/vk.xml

```diff
@@ -22179,6 +22180,12 @@ typedef void* <name>MTLSharedEvent_id</name>;
                 <enum bitpos="14" extends="VkExternalMemoryHandleTypeFlagBits" name="VK_EXTERNAL_MEMORY_HANDLE_TYPE_530_BIT_QNX"/>
             </require>
         </extension>
+        <extension name="VK_MSFT_extension_531" number="531" author="MSFT" contact="Jesse Natalie @jenatali" support="disabled">
```

Comment:
```suggestion
        <extension name="VK_MSFT_extension_531" number="531" author="MSFT" contact="Jesse Natalie @jenatali" supported="disabled">
```

This should resolve the CI problem.

Reply:
Whoops :smile: I apparently did a bad job copying.